go/consensus/tendermint/apps/beacon/state/state.go

```diff
@@ -23,6 +32,18 @@ var (
 	parametersKeyFmt = keyformat.New(0x41)
 )
 
+// ConsensusParmeters is the beacon consensus parameters.
+type ConsensusParameters struct {
+	Backend    string                     `json:"backend"`
```

Comment:
Why isn't the `Backend` field in `beacon.ConsensusParameters` and then this separate structure would not be needed?